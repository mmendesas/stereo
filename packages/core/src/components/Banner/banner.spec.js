import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Banner } from '.';

describe('Component | Banner', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Banner>Banner</Banner>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
