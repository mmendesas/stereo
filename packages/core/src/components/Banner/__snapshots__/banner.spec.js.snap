// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | Banner renders correctly 1`] = `
.c0 {
  height: 300px;
  background-color: #0aa;
  color: #fff;
  margin: 5px;
  border-radius: 5px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<section
  className="c0"
>
  Banner
</section>
`;
